pkg/cmd/cli/describe/describer.go

```diff
@@ -653,12 +653,12 @@ func (d *TemplateDescriber) DescribeObjects(objects []runtime.Object, out *tabwr
 		meta := kapi.ObjectMeta{}
 		meta.Name, _ = d.MetadataAccessor.Name(obj)
 		fmt.Fprintf(out, fmt.Sprintf("%s%s\t%s\n", indent, kind, meta.Name))
-		//meta.Annotations, _ = d.MetadataAccessor.Annotations(obj)
+		meta.Annotations, _ = d.MetadataAccessor.Annotations(obj)
```

Comment:
get rid of this, we should not be changing TemplateDescriber DescribeObjects at all